client/src/component/Pages.jsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import styles from '../componentsStyles/PagesStyle.jsx';
+
+
+class Pages extends React.Component {
```

Comment:
Since this component is stateless, I would refactor this to a functional stateless component by removing the constructor.

Reply:
I do plan on adding state to this once I add the search component